app/screens/settings-screen/account-screen.tsx

```diff
@@ -59,6 +59,15 @@ export const AccountScreen = ({ navigation }: Props) => {
   }
 
   const accountSettingsList: SettingRow[] = [
+    {
+      category: LL.common.accountLimits(),
+      id: "limits",
+      icon: "",
```

Comment:
What about some type of exclamation icon?  e.g. `exclamationcircleo`.  You can see the choices here https://oblador.github.io/react-native-vector-icons/.

Reply:
> What about some type of exclamation icon? e.g. `exclamationcircleo`. You can see the choices here https://oblador.github.io/react-native-vector-icons/.

👍